src/modules/graphics/opengl/Graphics.cpp

```diff
@@ -469,6 +469,12 @@ void Graphics::setActive(bool enable)
 	active = enable;
 }
 
+void Graphics::dispatch(int x, int y, int z)
+{
+	glDispatchCompute(x, y, z);
+	glMemoryBarrier(GL_ALL_BARRIER_BITS); // Strawman
```

Comment:
I'm fine with doing a proper synchronization implementation after this change is merged rather than before